config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -80,11 +66,53 @@ periodics:
             value: "us-west-2"
         securityContext:
           privileged: true
+        resources:
+          requests:
+            cpu: 1
+            memory: "4Gi"
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
     testgrid-tab-name: periodic-e2e-release-0-4
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
     testgrid-num-failures-to-alert: "2"
+- name: periodic-cluster-api-provider-aws-e2e-conformance-master
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  interval: 4h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-service-account: "true"
```

Comment:
Is this required?

Reply:
I don't *think* so, but it is set on the other jobs, so I left it for now.